src/index.js

```diff
@@ -0,0 +1,5 @@
+import types from './types'
+import infer from './infer'
+import Schema from './schema'
+
+module.exports = { infer, types, Schema }
```

Comment:
Would you like to have this change already? Or it can be made later on with other refactoring planned?